.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 
 jdk:
-  - oraclejdk7
```

Comment:
I built on Java 8 and ran tests with Java 7, and they pass. Not sure there's much else to do. (Though automating that pair of operations with Travis would be badass.)